topics/fair/tutorials/data-management/tutorial.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: tutorial_hands_on
+title: FAIR data management solutions
+zenodo_link: ''
+questions:
+- Is there a reproducibility crisis?
+- What can go wrong with data analysis?
+objectives:
+- Learn best practices in data management 
+- Learn how to introduce computational reproducibility in your research
+time_estimation: 10m
+key_points:
+- FAIR data management allows machines to automatically find and use the data accordingly.
+
+requirements:
+  -
+    type: "internal"
+    topic_name: fair-data
+    tutorials:
+      - fair-intro
+      
+tags:
+  - fair
+  - dmp
+  - data stewardship
+  
+contributors:
+- kkamieniecka
+- poterlowicz-lab
+priority: 2
+---
+
+
+# Introduction
+
+The FAIR (Findable, Accessible, Interoperable, Reusable)  data stewardship created the foundation for sharing and publishing digital assets, especially data. This apply to machine accessibility and emphasize that all digital assets should share data in a way that will enable maximum use and reuse. 
+
+This tutorial is a short introduction to the FAIR data management framework. You can find out more [here](https://elixiruknode.org/what-we-offer/fair-data-management/).
+
+> <agenda-title></agenda-title>
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Data management planning
+In recent years we have notice a data explosion. Number of sequence records in each release of [GenBank](https://www.ncbi.nlm.nih.gov/genbank/statistics/), from 1982 to the present, doubled in size approximately every 18 months. Great amounts of data available followed by expanding range of tools and computational solutions result in reproducibility crisis. Having **data management plan (DMP)** in place is essential to achieve FAIR data management. DMPs are often described as living documents and should be updated according to changing circumstances.
```

Comment:
you might be interested in the [abbreviations feature](https://training.galaxyproject.org/training-material/news/2021/05/25/abbreviations-tag.html) for DMP/FAIR